lib/html-proofer/utils.rb

```diff
@@ -34,5 +35,17 @@ def clean_content(string)
       end
     end
     module_function :clean_content
+
+
+    def init_stat
+      process = StatModule::Process.new('HTMLProofer')
+      process.version = "#{HTMLProofer::VERSION}"
+      process.description = 'If you generate HTML files, then this tool might be for you.'
+      process.maintainer = 'Garen Torikian'
+      process.email = 'gjtorikian@gmail.com'
+      process.website = 'https://github.com/gjtorikian/html-proofer'
```

Comment:
Please load these from the Gemfile itself, so that constants do not need to be recorded in two places.